### DISCUSSION TITLE
Add AlphaDroid to "main" Branch

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#18708] [General]
Hello @Apon77 ,

Could you please kindly add alphadroid to the supported roms? It's based on crdroid and the source is here

https://github.com/AlphaDroid-Project

Thank you for the time

**Comment by Apon77** [comment]
not popular or not active rom is not added, it is a very new rom, so it won't be added soon, need time.

**Reply by hipexscape** [reply]
Ok
